config.yaml

```diff
@@ -6,7 +6,10 @@ endpoints:
     conditions:
       - "[STATUS] == 200"
       - "[BODY].status == UP"
-      - "[RESPONSE_TIME] < 150"
+    ui:
+      badge:
+        response-time:
+          thresholds: [500, 850, 1350, 1650, 1750]
```

Comment:
As I consider this configuration a bit of an edge case, I'd rather not have it in the default `config.yaml`.